src/libs/actions/Report.js

```diff
@@ -55,6 +56,11 @@ const reportMaxSequenceNumbers = {};
 // Keeps track of the last read for each report
 const lastReadSequenceNumbers = {};
 
+// Map of temporary report action IDs a.k.a. optimistic
+// comments. These should be cleared when replaced by
```

Comment:
Can you please add to this comment to explain why it needs to be cleared, and why the recent fetch of report history is the best time to clear it?